### ISSUE TITLE
​UI is missing after disabling tracking

### ISSUE THREAD

**Issue by YourDaniel** [toeverything/AFFiNE#6842]
### What happened?

I updated Affine image to the newest version: 0.14.1 stable, went over to settings and toggled a button "Disable telemetry". After that the page reloaded but with no visible UI however everything in Networking tab returned 200 or 304.

### Distribution version

Web (app.affine.pro)

### What browsers are you seeing the problem on if you're using web version?

Chrome

### Are you self-hosting?

- [x] Yes

### Relevant log output

```shell
TypeError: Cannot read properties of undefined (reading 'opt_out_tracking_persistence_type')
    at eX.get_config (mixpanel.cjs.js:5789:26)
    at eX._gdpr_call_func (mixpanel.cjs.js:5913:34)
    at eX.opt_out_tracking (mixpanel.cjs.js:6020:10)
    at telemetry.tsx:10:7
    at uD (react-dom.production.min.js:243:332)
    at uK (react-dom.production.min.js:260:147)
    at e (react-dom.production.min.js:259:446)
    at e (react-dom.production.min.js:259:343)
    at react-dom.production.min.js:259:456
    at oD (react-dom.production.min.js:282:486)
a6 @ react-dom.production.min.js:188
t.callback @ react-dom.production.min.js:188
lY @ react-dom.production.min.js:156
uK @ react-dom.production.min.js:261
e @ react-dom.production.min.js:259
(anonymous) @ react-dom.production.min.js:259
oD @ react-dom.production.min.js:282
o_ @ react-dom.production.min.js:272
r4 @ react-dom.production.min.js:127
(anonymous) @ react-dom.production.min.js:282
oD @ react-dom.production.min.js:282
ow @ react-dom.production.min.js:268
x @ scheduler.production.min.js:13
T @ scheduler.production.min.js:14
(anonymous) @ jszip.min.js:13
d @ jszip.min.js:13
p @ jszip.min.js:13
postMessage (async)
n @ jszip.min.js:13
u.setImmediate @ jszip.min.js:13
u @ scheduler.production.min.js:14
T @ scheduler.production.min.js:14
(anonymous) @ jszip.min.js:13
d @ jszip.min.js:13
p @ jszip.min.js:13
postMessage (async)
n @ jszip.min.js:13
u.setImmediate @ jszip.min.js:13
u @ scheduler.production.min.js:14
M @ scheduler.production.min.js:14
n.unstable_scheduleCallback @ scheduler.production.min.js:18
ok @ react-dom.production.min.js:296
ob @ react-dom.production.min.js:265
o3 @ react-dom.production.min.js:307
o7.render.o9.render @ react-dom.production.min.js:309
(anonymous) @ index.tsx:64
(anonymous) @ index.tsx:69
t.a @ async module:49
18681 @ app-0d3d41fa.js:440
t @ bootstrap:19
t @ app-0d3d41fa.js:440
(anonymous) @ app-0d3d41fa.js:440
t.O @ chunk loaded:23
(anonymous) @ app-0d3d41fa.js:440
n @ jsonp chunk loading:72
(anonymous) @ app-0d3d41fa.js:1
Show 33 more frames
Show less
react-dom.production.min.js:127 Uncaught TypeError: Cannot read properties of undefined (reading 'opt_out_tracking_persistence_type')
    at eX.get_config (mixpanel.cjs.js:5789:26)
    at eX._gdpr_call_func (mixpanel.cjs.js:5913:34)
    at eX.opt_out_tracking (mixpanel.cjs.js:6020:10)
    at telemetry.tsx:10:7
    at uD (react-dom.production.min.js:243:332)
    at uK (react-dom.production.min.js:260:147)
    at e (react-dom.production.min.js:259:446)
    at e (react-dom.production.min.js:259:343)
    at react-dom.production.min.js:259:456
    at oD (react-dom.production.min.js:282:486)
```

**Comment by EYHN** [comment]
Please try clearing your browser's website storage to see if this can be restored.

We will look into this issue

**Comment by YourDaniel** [comment]
I hard reloaded the page, cleaned local storage and cache. Still the same error

**Comment by Tan5en5** [comment]
I've encountered the same issue 😟 

PS : I tried to clean the browser memory as explained by @EYHN , it worked, the UI is back 🥳 
But this means that it's impossible to disable telemetry for now 😞

**Comment by miladhatami1393** [comment]
I hard reloaded the page

**Comment by medinnna** [comment]
Same problem here. I also confirm that clearing local storage does the trick, hard reload wasn't enough.

**Comment by EYHN** [comment]
fixed in #6861
